txn/txn_test.go

```diff
@@ -689,7 +702,12 @@ func (s *S) TestPurgeMissingPipelineSizeLimit(c *C) {
 	c.Assert(err, IsNil)
 }
 
+var flaky = flag.Bool("flaky", false, "Include flaky tests")
+
 func (s *S) TestTxnQueueStressTest(c *C) {
+	if !*flaky {
```

Comment:
Perhaps include a comment as to why this test is flaky and how it fails when it flakes out?